src/pfe/file-watcher/server/src/controllers/projectStatusController.ts

```diff
@@ -523,27 +533,35 @@ function pingInTransitApplications(): void {
                                         projectID: projectID,
                                         appStatus: newState
                                     };
+                                    let newNotificationID = "";
                                     if (newState === AppState.started) {
                                         detailedAppStatus = {
                                             severity: "INFO",
                                             message: "",
-                                            notify: false
+                                            notify: false,
+                                            notificationID: newNotificationID
                                         };
                                         data.detailedAppStatus = detailedAppStatus;
                                         pingCountMap.delete(projectID);
                                     } else if (newState === AppState.starting && pingCount == pingCountLimit) {
                                         const troubleShootLinkLabel = await locale.getTranslation("projectStatusController.linkLabel");
+                                        if ( !oldNotificationID ) {
```

Comment:
Can probably simplify this block to `let newNotificationID = oldNotificationID ? oldNotificationID : crypto.randomBytes(16).toString("hex");